src/main/java/se/laz/casual/event/service/log/cli/log/ServiceLogger.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2024, The casual project. All rights reserved.
+ *
+ * This software is licensed under the MIT license, https://opensource.org/licenses/MIT
+ */
+
+package se.laz.casual.event.service.log.cli.log;
+
+import se.laz.casual.event.ServiceCallEvent;
+import se.laz.casual.event.service.log.cli.internal.EventServiceLoggerException;
+import se.laz.casual.event.service.log.cli.internal.StreamEncoder;
+import se.laz.casual.event.service.log.cli.runner.EventServiceLogParams;
+
+import java.io.File;
```

Comment:
For next version we could go with nio instead, it should be much more efficient for IO